app/views/docs/permissions.phtml

```diff
@@ -1,19 +1,19 @@
 <p>Appwrite permission mechanism offers a simple and yet flexible way to manage which users, teams, or roles can access a specific resource of your project, like documents and files.</p>
 
-<p>Using permissions, you can decide that only <span class="tag">user A</span> and <span class="tag">user B</span> will have read access to a specific database document, while <span class="tag">user C</span> and <span class="tag">team X</span> will be the only ones with write access.</p>
+<p>Using permissions, you can decide that only <span class="tag">user A</span> and <span class="tag">user B</span> will have read and update access to a specific database document, while <span class="tag">user C</span> and <span class="tag">team X</span> will be the only ones with create and delete access.</p>
 
-<p>As the name suggests, read permission will allow users and teams to view a resource while with write permission, they will be able to both update or delete it.</p>
+<p>As the name suggests, read permission allows a user to view a resource, create allows users to create new resources, update allows a user to make changes to a resource, and delete allows the user to remove the resource.</p>
 
-<p>Read and write permissions can be given to specific users, entire teams, or only to a particular group of role members inside a team.</p>
+<p>All permissions can be given to specific users, entire teams, or only to team members with a specific role.</p>
```

Comment:
We are missing that it cal also be set to stuff like `any`, `guests` or `users`. How would we call these.. groups of users?